app/Checks.php

```diff
@@ -26,9 +26,9 @@
 
 use App\Models\Device;
 use App\Models\Notification;
-use Auth;
 use Cache;
 use Carbon\Carbon;
+use Illuminate\Support\Facades\Auth;
```

Comment:
This one is wrong, isn't it?

Reply:
Why?

Reply:
Facades are not meant to be access directly, but through the alias (\Auth for this).

Reply:
Hmmm didn't know.
It works bad with phpstan tho.

https://github.com/nunomaduro/larastan/issues/506#issuecomment-602792071

Reply:
Also. Everything here use it directly; 
https://laravel.com/docs/8.x/authentication#introduction